common/constants/default.go

```diff
@@ -10,7 +10,7 @@ import (
 const (
 	DefaultKubeContentType         = "application/vnd.kubernetes.protobuf"
 	DefaultKubeNamespace           = v1.NamespaceAll
-	DefaultKubeQPS                 = 100.0
+	DefaultKubeQPS                 = 5.0
```

Comment:
could you please explain why it was changed to 5 ?

Reply:
https://github.com/kubernetes/kubernetes/blob/1e7c07ac4ea28587551c1e766b16d1eeb834185f/staging/src/k8s.io/client-go/rest/config.go#L44

Reply:
The qps should less than the burst.
In keeping with [client-go](https://github.com/kubernetes/kubernetes/blob/1e7c07ac4ea28587551c1e766b16d1eeb834185f/staging/src/k8s.io/client-go/rest/config.go#L44).

Reply:
EdgeController is an aggregation of shadow management for pods, nodes, configmaps etc. at edge. The default value 5 in k8s client-go is for a single client e.g. kubectl, which is quite different with EdgeController load here.

Reply:
Would recommand increase burst instea.
And it would be even better if you can provide test result comparison between before and after.

Reply:
@kevin-wangzefeng set the burst to 200, which is twice of qps, is than ok?